src/rust/include/rust/VA_OPT.hpp

```diff
@@ -0,0 +1,152 @@
+//  Copyright (c) 2019 Will Wray https://keybase.io/willwray
```

Comment:
```suggestion
//  Copyright (c) 2019 Will Wray https://keybase.io/willwray
//  Copyright (c) 2020 The Zcash developers
```

Reply:
I've made no changes to this file, so I'm not sure we can add this yet.

Reply:
Ah, I see you've also suggested changes.